panther_gazebo/README.md

```diff
@@ -0,0 +1,69 @@
+
+- `simulate_discharging` [*bool*, default: **false**]: Enables battery simulation. If set to `true`, the battery will discharge **at a constant rate** (regardless of joint torque), and if it depletes completely, the robot will stop moving. When set to `false`, the battery will not discharge, but the battery status information will still be published on the `/battery` topic.
+- `initial_charge_percentage` [*float*, default: **70.0**]: Sets the initial charge percentage of the battery.
+- `charging_time` [*float*, default: **6.0**]: Specifies the charging time for the battery in hours.
+- `power_load` [*float*, default: **120.0**]: Represents the average power load during normal operation **[W]** and is initially set to 120.0 W. With the default power load of 120.0 W, the robot can operate for up to 8 hours. When the `simulate_discharging` parameter is set to `false`, this value defaults to 0.0 W. Users are encouraged to customize this value to match their specific requirements. For more information on Panther power consumption, please refer to [Panther Battery & Charging Documentation](https://husarion.com/manuals/panther/#battery--charging).
+
+#### Charging Process
+
+Unfortunately, there is no straightforward way to exchange `LinearBatteryPlugin` services between ROS and Gazebo Transport, so you need to use the `ign` commands. As a result, the method of charging differs between the real and simulated robot.
+
+You can start the charging process by calling the Ignition service:
+
+```bash
+ign service --service /model/panther/battery/panther_battery/recharge/start --reqtype ignition.msgs.Boolean --reptype ignition.msgs.Empty --req '' --timeout 0
+```
+
+and stop it using:
+
+```bash
+ign service --service /model/panther/battery/panther_battery/recharge/stop --reqtype ignition.msgs.Boolean --reptype ignition.msgs.Empty --req '' --timeout 0
+```
+
+### Use of GPS in Simulation
+
+The NavSat system is utilized to publish the Panther robot's position within the Gazebo world. It manages navigation satellite sensors, such as GPS, which report position and velocity in spherical coordinates (latitude/longitude) through Ignition Transport.
+
+The NavSat sensors depend on the spherical coordinates of the world origin being configured. This configuration can be accomplished, for instance, by employing the `<spherical_coordinates>` tag within the world's SDF or by utilizing the Ignition `/world/world_name/set_spherical_coordinates` service.
+
+To obtain GPS data in Ignition, all you need to do is include the [external_antenna](../panther_description/urdf/components/external_antenna.urdf.xacro) macro to your robot model and add the following tag to your world's SDF file and point this file with `world` param:
```

Comment:
Missing info that it is also required to modify gz_bridge to sent NavSat topic. Also, I would add info that default world is already configured to support GPS data, If user want's to use custom world then he needs to add spherical_coordinates.